config/fcos/v1_6_exp/translate.go

```diff
@@ -367,3 +375,75 @@ func buildGrubConfig(gb Grub) string {
 	superUserCmd := fmt.Sprintf("set superusers=\"%s\"\n", strings.Join(allUsers, " "))
 	return "# Generated by Butane\n\n" + superUserCmd + strings.Join(cmds, "\n") + "\n"
 }
+
+func (c Config) handleSelinux(options common.TranslateOptions) (types.Config, translate.TranslationSet, report.Report) {
+	rendered := types.Config{}
+	ts := translate.NewTranslationSet("yaml", "json")
+	var r report.Report
+
+	hasValidModule := false
+
+	for _, module := range c.Selinux.Module {
+		if isValidModule(module) {
+			hasValidModule = true
```

Comment:
we can get rid of this. as we can look at our rendered types to see if there was a valid module